pkg/lang/golang/aws_runtime/aws.go

```diff
@@ -47,10 +48,12 @@ func (r *AwsRuntime) AddExecRuntimeFiles(unit *core.ExecutionUnit, result *core.
 		ExecUnitName:   unit.Name,
 	}
 
-	err := golang.AddRuntimeFile(unit, templateData, "Dockerfile", DockerFile)
-	if err != nil {
-		return err
+	if runtime.ShouldOverrideDockerfile(unit) {
+		err := golang.AddRuntimeFile(unit, templateData, "Dockerfile", DockerFile)
```

Comment:
Should this also set the `unit.DockerfilePath` to "Dockerfile", or is that not necessary

Reply:
the ShouldOverrideDockerfile method is what sets that since its not language specific

Reply:
It looks like it only sets it if `ShouldOverrideDockerfile` is false.

Reply:
right because it will be "" if we override the dockerfile since the dockerfile will be at the root of the execution units directory